internal/connector/connector.go

```diff
@@ -124,12 +124,7 @@ func Run(ctx context.Context, options Options) error {
 	logging.L.Debug().Str("uniqueID", checkSum).Msg("Cluster uniqueID")
 
 	if options.Name == "" {
-		autoname, err := k8s.Name(checkSum)
-		if err != nil {
-			logging.Errorf("k8s name error: %s", err)
-			return err
-		}
-		options.Name = autoname
+		return fmt.Errorf("destination name is required")
```

Comment:
Should we give a name to this connector automatically rather than making this as a breaking change? We may be able to do it with simpler logic than the `k8s.Name()` function.